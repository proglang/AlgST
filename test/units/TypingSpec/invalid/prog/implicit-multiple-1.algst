-- TODO: the error should mention where these are defined if they come from the
-- current module.

{a} : Int
{a} = 10

{b} : Int
{b} = 20

c : Int
c = _
