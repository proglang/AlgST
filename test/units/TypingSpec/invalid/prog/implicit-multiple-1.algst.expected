
11:5: error:
	 Multiple chocies for implicit of type Int
	   • a
	     imported from M
	   • b
	     imported from M